cmd/archer/template/template.go

```diff
@@ -1,19 +1,23 @@
+// +build !windows
+
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
 // Package template provides usage templates to render help menus.
 package template
 
 import (
-	"strings"
+	"fmt"
 
-	"github.com/fatih/color"
 	"github.com/spf13/cobra"
+
+	"github.com/aws/PRIVATE-amazon-ecs-archer/internal/pkg/cli/group"
 )
 
 // RootUsage is the text template for the root command.
-const RootUsage = `{{h1 "Commands"}}{{ $cmds := .Commands }}{{$groups := mkSlice "Getting Started ✨" "Develop 🔧" "Settings" }}{{range $group := $groups }}
-  {{h2 $group}}{{range $cmd := $cmds}}{{if isInGroup $cmd $group}}
+var RootUsage = fmt.Sprintf("{{h1 \"Commands\"}}{{ $cmds := .Commands }}{{$groups := mkSlice \"%s\" \"%s\" \"%s\" }}{{range $group := $groups }} \n",
```

Comment:
nit: another alternative is to set `cmd.Annotations["groups"] = []string{group.GettingStarted, group.Develop, group.Settings}` in main.go. This way we can replace `{{$groups := mkSlice \"%s\" \"%s\" \"%s\" }}` with `{{$groups := (index .Annotations "groups"}}`

I find it slightly neater since we can keep the text as a constant, and get rid of fmt.Sprintf